NerdDinner/Models/DinnerRepository.cs

```diff
@@ -21,10 +21,9 @@ public IQueryable<Dinner> FindAllDinners()
         // Retrieves all dinners after the the current date & time
         public IQueryable<Dinner> FindUpcomingDinners()
         {
-            return from dinner in db.Dinners
-                   where dinner.EventDate > DateTime.Now
-                   orderby dinner.EventDate
-                   select dinner;
+            return db.Dinners
```

Comment:
Changed from LINQ query to LINQ extension method. The extension methods are more modern, easier to use, and generally preferred over the LINQ queries.